src/App.vue

```diff
@@ -1,8 +1,18 @@
 <template>
-  <div id="nav"></div>
-  <router-view />
+  <div id="nav">
+    <BlogCategory category="NATURE" />
+    <router-view />
+  </div>
 </template>
 
+<script>
+import BlogCategory from './components/BlogCategory'
+
+export default {
+  components: { BlogCategory },
+}
+</script>
+
```

Comment:
Please remove the changes made to this file. This is because this component won't be directly imported into the App.vue